Products/CMFPlone/static/components/ace-builds/kitchen-sink.html

```diff
@@ -11,10 +11,10 @@
       commit  
     -->
 
-    <link rel="stylesheet" href="demo/kitchen-sink/styles.css" type="text/css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="demo/kitchen-sink/styles.css" media="screen" charset="utf-8">
     <script async="true" src="https://use.edgefonts.net/source-code-pro.js"></script>
 
-    <link href="./doc/site/images/favicon.ico" rel="icon" type="image/x-icon">
```

Comment:
you can't remove the type attribute for the favicon.

Reply:
ooh why is it so? it is also in the <link> tag.

Reply:
because this is a different `link` type:

https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types